checkers/model_form.py

```diff
@@ -0,0 +1,55 @@
+import ast
+
+from .checker import Checker
+from .issue import Issue
+
+
+class DJ06(Issue):
+    code = 'DJ06'
+    description = 'ModelForm.Meta should not set exclude, set fields instead'
```

Comment:
nitpicking alert: what about putting quotes around the arguments names?

```python
description = 'ModelForm.Meta should not set "exclude", set "fields" instead'
```

Reply:
sounds good 👍